### DISCUSSION TITLE
Adding a New Field to Certificate

### DISCUSSION THREAD

**Discussion by mahajanmahesh935** [Sunbird-RC/community#1056] [Questions (FAQs)]
Hi Team,

I need assistance with adding a new field to the certificate beyond the default fields (recipientName, trainingName, issuanceDate).

We attempted the following approaches:

1. Passing a new field in the certificate SVG as {{credentialSubject.recipientOrg}}, but it didn’t work.
2. Including the org variable in the certificate creation API, which also didn’t yield the desired results.

Could someone please guide us on how to properly add a custom field (e.g., recipientOrg) to the certificate? Any insights or documentation links would be greatly appreciated.

CC : @komalm @paritshivani @Abhilash-Kishor @vinod303

**Comment by mahajanmahesh935** [comment]
Hi @pallakartheekreddy,
We had a call on Friday (29 dec) regarding this ticket. As per our discussion, you mentioned that you would get back to us with a solution.

I was wondering if you’ve had a chance to look into it. Please let me know if any further information or clarification is needed from my side.

Thank you!

**Comment by mahajanmahesh935** [comment]
Hi @pallakartheekreddy,

I’m following up on the ticket we discussed during our call on Friday. As per our conversation, you mentioned that you would provide a solution.

This has become a blocker for us, and resolving it is quite urgent. Could you please provide an update or let me know if any additional input is required from my side to expedite the resolution?

Looking forward to your response.

**Reply by pallakartheekreddy** [reply]
@mahajanmahesh935 We have verified this, and it seems these changes need to be made in the Ed/Lean building block since it is being used in the course progress workflow. Could you please create a thread for this in the Ed discussion forum?

**Comment by mahajanmahesh935** [comment]
Moved this ticket to Ed discussion forum https://github.com/orgs/Sunbird-Ed/discussions/755
